apps/calendar/js/views/hour_double_tap.js

```diff
@@ -40,12 +40,40 @@ HourDoubleTap.prototype = {
     this.alldaysHolder.removeEventListener('click', this._onAllDayTap);
   },
 
-  _onDayTap: function(evt) {
+  _isEventRightOnButton(evt) {
     var target = evt.target;
     if (!target.classList.contains('md__day')) {
+      // Could be aggressive event fuzzing registering a tap on an existing
+      // event or the "add event" UI, where the tap is really below that item.
+      // See bug 1210201.
+
+      // In this case, offsetY is the vertical space between the top of the
+      // element and the actual tap, so if the tap is bigger than the height
+      // of the element, treat it as an md__day tap.
+      var tappedOnExistingEvent =
```

Comment:
The last part of this comment can be removed: this function is just focused on if it is an event button.